### PR TITLE
Add formatting for checksums in draft release body

### PR DESCRIPTION
**Pull request by praveenrewar**
#### What this PR does / why we need it:
- Add formatting for checksums in draft release body
- Add checksums.txt file as asset
- Add `./` in the checksums.txt

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

##### Review Checklist:

- [ ] Follows the [developer guidelines](https://carvel.dev/shared/docs/latest/development_guidelines/)
- [ ] Relevant tests are added or updated
- [ ] Relevant docs in this repo added or updated
- [ ] Relevant carvel.dev docs added or updated in a separate PR and there's
  a link to that PR
- [ ] Code is at least as readable and maintainable as it was before this
  change

#### Additional documentation e.g., Proposal, usage docs, etc.:

```docs

```

### CODE REVIEW

File: .github/workflows/release-process.yml [hunk 1]
```diff
@@ -37,17 +37,22 @@ jobs:
           ./hack/build-release.sh
           cd cli
           ./hack/build-binaries.sh
-          shasum -a 256 kctrl-* | tee -a ../tmp/checksums.txt
+          shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
+          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
+          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
```
Comment (praveenrewar) [inline .github/workflows/release-process.yml]: I tried something like `{ echo "# :open_file_folder: Files Checksum"; cat ../tmp/checksums.txt } tee ../tmp/checksums-formatted.txt` to make it inline, but it didn't work expectedly.

File: .github/workflows/release-process.yml [hunk 2]
```diff
@@ -37,17 +37,22 @@ jobs:
...
         uses: softprops/action-gh-release@17cd0d34deddf848fc0e7d9be5202c148c270a0a
         with:
           name: ${{ github.ref_name }}
           token: ${{ secrets.GITHUB_TOKEN }}
-          body_path: ./tmp/checksums.txt
+          body_path: ./tmp/checksums-formatted.txt
           files: |
             ./cli/kctrl-*
             ./tmp/release.yml
+            ./tmp/checksums.txt
```
Comment (cppforlife) [inline .github/workflows/release-process.yml:55]: whats the rationale for including checksums.txt artifact?

Comment (praveenrewar) [reply .github/workflows/release-process.yml:55]: It's nice to have for something machine readable/automations. We did once receive a request from a user for `kapp`. All the other repos also have it.

File: .github/workflows/release-process.yml (hunk 1 above)
Comment (neil-hickey) [reply .github/workflows/release-process.yml]: nit: You could replace the escape characters of your echo with

echo '```'

It's a little more readable to me. But this ain't a blocker :)

Comment (praveenrewar) [reply .github/workflows/release-process.yml]: > echo '```'

Totally makes sense.

Comment (praveenrewar) [reply .github/workflows/release-process.yml]: Done

**Review by neil-hickey — APPROVED** [review]
LGTM

**Pull request merged**